### ISSUE TITLE
Api module list not showing relationships

### ISSUE THREAD

**Issue by patrickstump** [salesagility/SuiteCRM#7094]
If Module B contains a many to one relationship with Module A, when using the API module list, the relationship shows up as many blank spaces instead of the content.
This does not appear to be occurring when requesting a specific record, only when requesting a list of records. 

#### Issue
When using the Api to request of list of records in a module, it is not returning relationship information correctly in the returned data.
**Example:**
Account "test account" has a contact "test contact" associated with it.
Pull the list of contacts `https://127.0.0.1/Api/V8/module/Contacts`
If you scroll through the results to find "account_name" and other related fields, you will see many spaces instead of actual content.
Please note to scroll the block code to the right to see that it goes for a while with the blank space.
```
                "lead_source": "",
                "account_name": "                                                                                                                                                                                                                                                              ",
                "account_id": "                                    ",
                "opportunity_role_fields": "                                                                                                                                                                                                                                                              ",
                "opportunity_role_id": "",
```
for reference, when you call the record by id `https://127.0.0.1/Api/V8/module/Contacts/aa29eebf-73fe-0c24-41a0-5c9771600d16`
you receive:
```
            "lead_source": "",
            "account_name": "test account",
            "account_id": "6add770a-7f5d-28c3-1304-5c91070821a3",
            "opportunity_role_fields": "",
            "opportunity_role_id": "",
```

#### Expected Behavior
Listing Module records via the /Api/V8/module/{module} should include the same data in the list as calling with the record id /Api/V8/module/{module}/{id}.
```
            "account_name": "test account",
            "account_id": "6add770a-7f5d-28c3-1304-5c91070821a3",
```
#### Actual Behavior
The record data in the list does not include related record information as it does in the individual id request.
```
                "account_name": "                                                                                                                                                                                                                                                              ",
                "account_id": "                                    ",
```
#### Possible Fix
Unknown 

#### Steps to Reproduce

1. Create an Account Record "test account"
2. In contacts under "test account" create a contact "test contact"
3. Pull the information to list the contacts `https://127.0.0.1/Api/V8/module/Contacts`
4. To see the data showing up in the individual record pull it by id `https://127.0.0.1/Api/V8/module/Contacts/{id}` it will show up.

#### Context
In interacting with the api, I cannot create the full records in my code currently.  Instead of retrieving a list of full records, I need to
1. retrieve the list of records
2. get the record Ids from the list
3. Retrieve every module from the API individually with a call for each.
Effect: Instead of one call to the API . I need to have `1+N` Calls where N is the number of records returned.

#### Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* SuiteCRM Version used: 7.11.2
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Python 3.7.x and Postman 7.0.6
* Environment name and version (e.g. MySQL, PHP 7): Mariadb and php7
* Operating System and version (e.g Ubuntu 16.04): ubuntu 18.04

**Comment by Dillon-Brown** [comment]
Fixed by #7147

**Comment by piotrkowalskipl** [comment]
I did update to the latest release 7.11.5, but the problem seems still to exist.

See my [gist](https://gist.github.com/piotrkowalskipl/4d2826caa67e45987777f1c0d46a127f) with curl examples.

Can some someone confirm if the latest upgrade fixed the case?
